app/services/audit_results.rb

```diff
@@ -165,10 +165,23 @@ def report_results(logger=Rails.logger)
       end
       send_honeybadger_notification(r) if HONEYBADGER_REPORT_CODES.include?(r.keys.first)
     end
+    update_moab_status_details(complete_moab) if complete_moab
     report_errors_to_workflows(workflow_results)
     result_array
   end
 
+  # expects complete_moab to not be nil
+  def update_moab_status_details(complete_moab)
+    moab_error_codes = WORKFLOW_REPORT_CODES + [INVALID_MOAB]
+    moab_error_results = result_array.select { |r| moab_error_codes.include?(r.keys.first) }
+
+    # transaction_ok = ActiveRecordUtils.with_transaction_and_rescue(moab_error_results) do
```

Comment:
I wasn't sure whether to wrap this in a transaction or not.  I leaned towards "not."  If you agree, I will remove the comments here.  If you disagree, I can uncomment what is here.